supportedBackends/google/pipelines/common/src/test/scala/cromwell/backend/google/pipelines/common/PipelinesApiAsyncBackendJobExecutionActorSpec.scala

```diff
@@ -365,6 +365,21 @@ class PipelinesApiAsyncBackendJobExecutionActorSpec extends TestKitSuite("JesAsy
     retryableHandle.throwable.getMessage should include("will be restarted with another preemptible VM")
   }
 
+  it should "treat a PAPI v2 style error message as preemptible if the VM was preemptible" in {
+    val actorRef = buildPreemptibleTestActorRef(1, 2)
+    val jesBackend = actorRef.underlyingActor
+    val runId = StandardAsyncJob(UUID.randomUUID().toString)
+    val handle = new JesPendingExecutionHandle(null, runId, None, None)
+
+    val failedStatus = UnsuccessfulRunStatus(Status.ABORTED, Option(PipelinesApiAsyncBackendJobExecutionActor.FailedV2Style), Seq.empty, Option("fakeMachine"), Option("fakeZone"), Option("fakeInstance"), wasPreemptible = true)
+    val executionResult = jesBackend.handleExecutionResult(failedStatus, handle)
```

Comment:
This method calls a `StandardExecutionActor` method, which indirectly needs to look up stderr and rc files (where it gets them from, I cannot tell...) - and that's why it returns a `Future` rather than just a value.

TOL: is it simple and easy to retarget this test to only test the functionality that is (a) deterministic and (b) changed by this PR?

Reply:
This is just a copy/paste of an existing test adapted for a new preemptible mode. If we really want to make these changes I think that should be a separate ticket.